pkg/controllers/scheduler/handle.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2023 The KubeAdmiral Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// The design and implementation of the scheduler is heavily inspired by kube-scheduler and karmada-scheduler. Kudos!
+
+package scheduler
+
+import (
+	"github.com/kubewharf/kubeadmiral/pkg/controllers/scheduler/framework"
+	"k8s.io/client-go/dynamic"
+)
+
+func (s *Scheduler) buildFrameworkHandle() framework.Handle {
```

Comment:
nit: why is this a method on `Scheduler`?

Reply:
@gary-lgy because it is an uti to read many misc fields from the `Scheduler`. The whole point of this method is to translate fields from `*Scheduler` to the methods in `framework.Handle`.

Reply:
Just curious, why not define the constructor as a standalone function? If we define it as a method on scheduler, we cannot instantiate a `Handle` outside the scheduler.

Reply:
Actually I am considering if I could just make the `*handle` contain a field that references `*Scheduler`, then just multiplex to the fields in the impl methods.

Reply:
I was concerned if there is a use case in the future (especially after @limhawjia implements dynamic scheduling profiles) where the handle fields would need to be dynamically updated and can only be fetched during actual use.

Reply:
@gary-lgy bump